src/DynamoCore/Scheduler/UpdateRenderPackageAsyncTask.cs

```diff
@@ -170,6 +170,12 @@ private void GetRenderPackagesFromMirrorData(MirrorData mirrorData, bool display
                 try
                 {
                     graphicItem.Tessellate(package, factory.TessellationParameters);
+                    if (package.MeshVertexColors.Count() > 0
+                        || package.LineStripVertexColors.Count() > 0
+                        || package.PointVertexColors.Count() > 0)
+                    {
+                        package.RequiresPerVertexColoration = true;
+                    }
```

Comment:
`RequiresPerVertexColoration` property is used only for meshes so I think you can remove the check for lines and points.